### ISSUE TITLE
Should we not remove `resolved_early` from `ResolveProposal` event because `multisig_voting` don't have concept of `early_resolution_vote_threshold`?

### ISSUE THREAD

**Issue by vpanchal-supra** [Entropy-Foundation/aptos-core#70]
As `multisig_voting` module does not have a concept of `early_resolution_vote_threshold` and it only considers `min_vote_threshold` so should we not remove  `resolved_early` from `ResolveProposal`.
- Right now every time `ResolveProposal` event is emitted with `resolved_early=true` because right now proposal with more than  `min_vote_threshold` is considered as early proposal.

**Comment by sjoshisupra** [comment]
@vpanchal-supra agreed. Assigning this to @nizam-supraoracles , however, whether this should be part of tge would have to be coordinated with @arundsupra depending upon code freeze date. As it is, this is merely a cleanup and not a critical bug.

**Comment by vpanchal-supra** [comment]
Yes Dr. @sjoshisupra, This is just an optimization related issue.

**Comment by nizam-supraoracles** [comment]
Dr. @sjoshisupra, in this case, do we need the `can_be_resolved_early` function?

**Comment by nizam-supraoracles** [comment]
As discussed with Dr. @sjoshisupra, this issue is still under debate due to the use of the `can_be_resolved_early` function. We're unsure if the `resolved_early` field will be required by the frontend or somewhere in governance dashboard. Therefore, we can leave this issue unresolved for now until we confirm whether this information is needed by the governance dashboard

**Comment by nizam-supraoracles** [comment]
@sjoshisupra @vpanchal-supra, Just checking if this issue is still relevant. It has been on hold for the last few months, and we can close it if it’s no longer needed.
